### PR TITLE
Account command that shows and updates user profile information

### PR DESCRIPTION
**Pull request by diegoribero**
Adds **matrix account** to fetch user data and **matrix account profile** to set/update it.

This could use some improvements on a following update (data isn't pre-loaded on the input fields, think we can do that if we change prompt for inquirer, need to test)

_This PR requires the **matrix-firebase** PR **[user.get and user.update](https://github.com/matrix-io/matrix-firebase/pull/13)** to be merged for the commands to work properly._

### CODE REVIEW

**Comment by eighteyes** [comment]
Output from matrix account profile

```
Developer info -- Name SeanC
Developer info -- URL (Optional) http://admobilize.com
Developer info -- URL (Optional) http://admobilize.com
```
Note double URL

**Comment by eighteyes** [comment]
```
{ name: 'SeanC',
  url: 'http://admobilize.com',
  repo: 'http://admobilize.com',
  track: 'Y' }
PROFILE:  { data:
   { name: 'SeanC',
     url: 'http://admobilize.com',
     githubUrl: 'http://admobilize.com',
     trackOk: 'Y' } }
```
shows after profile. shouldn't expose raw datas.

File: bin/matrix-account.js [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+  Matrix.validate.user();
+  Matrix.loader.start();
+  var target = Matrix.pkgs[0];
+
+  if (Matrix.pkgs.length > 1) { //If more than one arg, incorrect format
+    displayHelp();
+  } else {
+    Matrix.firebaseInit(function () {
+      if (Matrix.pkgs.length === 1 && target === 'profile') { //If asking for profile specifically
+        async.waterfall([
```
Comment (eighteyes) [inline bin/matrix-account.js]: oooh composition 👍

File: bin/matrix-account.js [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+      pattern: /[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)/,
+      message: 'Must provide a valid url',
+      description: 'URL (Optional)',
+    }
+
+    var repoP = {
+      name: 'repo',
+      pattern: /[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)/,
+      message: 'Must provide a valid url',
+      description: 'URL (Optional)',
```
Comment (eighteyes) [inline bin/matrix-account.js]: Github URL

**Comment by eighteyes** [comment]
Output, I updated `matrix-firebase`

```
PROMPT CALLBACK!!
 REGISTER
 --
err: null out: { access_token: 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ2IjowLCJpYXQiOjE0ODMxMTI1NDksImQiOnsidXNlclRva2VuIjp0cnVlLCJ1aWQiOiI1ODY2ODA2NTZlYThlODIwMDAyMzFjZmQifSwiZXhwIjoxNDgzMTQ4NTQ5LCJpc3MiOiJodHRwOi8vZGV2LWRlbW8uYWRtb2JpbGl6ZS5jb20ifQ.XhF7kAETEiBydiKk7TpFD4DzvBUQmdeywFw6bqPBHi4',
  expires_in: 1483148549,
  role: 'Business',
  token_type: 'bearer',
  message: 'User created successfully.' }
--`
```

**Comment by diegoribero** [comment]
Got rid of errors, cleaned up messy logs, adjusted so it asks for GitHub User instead of URL.

**Comment by eighteyes** [comment]
Loooooks goood. Trying to clean up tests.

**Pull request merged**